packages/shopify-codemod/README.md

```diff
@@ -11,12 +11,32 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 
 ## Included Transforms
 
+### `conditional-assign-to-if-statement`
+
+Changes conditional assignment of default values to if statements (see [`no-unused-expressions`.](http://eslint.org/docs/rules/no-unused-expressions))
+
+```sh
+jscodeshift -t shopify-codemod/transforms/conditional-assign-to-if-statement <file>
+```
+
+#### Example
+
+```js
+foo || (foo = 'bar');
+
+// BECOMES:
+
+if (!Boolean(foo)) {
```

Comment:
Talked it over on Slack, @GoodForOneFare let's go with the simpler `if (!foo.bar) {}` conditions. Made an issue to clarify this in the styleguide.